src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements AreaCalculetion {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color, String name) {
+        this.secondLeg = secondLeg;
+        this.firstLeg = firstLeg;
+    }
+
+    public double getFirstLeg() {
+        return firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return getFirstLeg() * getSecondLeg() * COEFFICIENT;

```

Comment:
The variable 'COEFFICIENT' is not defined anywhere in the class. According to the checklist, all magic numbers should be constants with descriptive names.